doc/Makefile

```diff
@@ -23,11 +23,12 @@ help:
 DOCUMENTER_OPTIONS := linkcheck=$(linkcheck) doctest=$(doctest) buildroot=$(call cygpath_w,$(BUILDROOT)) \
     texplatform=$(texplatform)
 
-UnicodeData.txt:
-	$(JLDOWNLOAD) http://www.unicode.org/Public/9.0.0/ucd/UnicodeData.txt
+$(SRCCACHE)/UnicodeData.txt:
```

Comment:
Since this change is in doc and not in deps, I don't think there is a SRCCACHE variable:
```
~/julia/doc$ make print-SRCCACHE
SRCCACHE=
```